cmd/kube-proxy/app/server.go

```diff
@@ -203,6 +203,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.config.EnableProfiling, "profiling", o.config.EnableProfiling, "If true enables profiling via web interface on /debug/pprof handler.")
 
 	fs.Float32Var(&o.config.ClientConnection.QPS, "kube-api-qps", o.config.ClientConnection.QPS, "QPS to use while talking with kubernetes apiserver")
+	fs.Var(&o.config.DetectLocalMode, "detect-local-mode", "Mode to use to detect local traffic")
```

Comment:
It was originally detect-local, but -mode got added as it one of the earlier reviews said it was more expressive. I'm happy to change it back to detect-local if we feel that is better.

Reply:
My earlier comment suggested `--detect-local-mode` since I instinctively assume `--detect-local` is a bool flag. It's more clearer to me that `--detect-local-mode` requires a string value.

Reply:
OK, I see that reasoning, and I don't care to spend any further energy on it. :)